### PR TITLE
Add a Swift package manifest for easy universal 2 builds

### PR DESCRIPTION
**Pull request by wbond**
Thanks for this tool!

Wanted to see if support for the SPM would be of benefit to users, hopefully making it easier to compile universal 2 builds without having to compile the architectures separately using `-target` and then `lipo`ing them together.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// swift-tools-version:5.5
```
Comment (saagarjha) [inline Package.swift:1]: Could you please format this file? The indentation isn't consistent. I've been using this with swift-format for the rest of the project, for reference:

```json
{
	"indentation": {
		"tabs": 1
	},
	"indentSwitchCaseLabels": true,
	"lineLength": 10000,
	"tabWidth": 4
}
```

I should probably put this in the repo, to be honest…

**Review by saagarjha — CHANGES_REQUESTED** [review]
I kind of wish swiftc would accept multiple -target flags, but alas…

Anyways, this looks mostly fine to me, but a couple nits.

File: README.md [hunk 2]
```diff
@@ -2,9 +2,19 @@
...
 unxip is a command line-tool designed for rapidly unarchiving Xcode XIP files and writing them to disk with good compression. Its goal is to outperform Bom (which powers `xip(1)` and Archive Utility) in both performance and on-disk usage, and (at the time of writing) does so by a factor of about 2-3x in time spent decompressing and about 8% in space.
 
-## Installation
+## Compiling
 
-Not much installation is needed to use unxip: simply download unxip.swift and compile it using `swiftc -parse-as-library -O unxip.swift` to produce the `unxip` binary.
+To build a Universal 2 (arm64/x86_64) executable, run:
+
+```bash
+swift build -c release --arch arm64 --arch x86_64
```
Comment (saagarjha) [inline README.md:10]: GitHub seems to have eaten this somehow…but anyways I prefer this a little better.

```suggestion
```console
$ swift build -c release --arch arm64 --arch x86_64
```

File: Package.swift (hunk 1 above)
Comment (wbond) [reply Package.swift:1]: Awesome, I:

 - Added a config and readme about this
 - Formatted Package.swift

File: README.md (hunk 2 above)
Comment (wbond) [reply README.md]: Done

**Review by saagarjha — APPROVED** [review]

**Comment by saagarjha** [comment]
I think swift build is common enough that it's probably fine for this. For local development I just use Sublime Text and swiftc from the command line, since I don't really need to build it as a Universal binary to run it on my machine. I was expecting this to be true for most users too; do you have a particular reason you're building for both?

**Comment by wbond** [comment]
I’m building it to distribute in binary form to our mixed architecture of iOS engineers. On my M1 it seems to reduce extraction time from 307s to 154s for Xcode 13.3, which is awesome!

**Comment by saagarjha** [comment]
Cool, glad you're finding it useful!

**Comment by saagarjha** [comment]
FYI, I partially reverted the README change because apparently SPM requires Xcode to be installed (for xctest? idk) and that's not ideal for a tool meant to grab a new Xcode. The instructions to build a Universal binary are still there, but I moved it down.

**Pull request merged**